### PR TITLE
Remove Oauth2 service, example, package

### PR DESCRIPTION
**Pull request by tomaszdurka**
- Remove oauth from docker-compose
- Remove oauth-example
- Remove oauth2 create-gke deployment
- Remove oauth2 package
- Revert mistake port change

### CODE REVIEW

File: config/create-gke-manifest.js [hunk 1]
```diff
@@ -659,68 +656,6 @@ async function applyApiServer() {
                     }
                   }
                 },
-                {
-                  name: 'OAUTH_GOOGLE_DISCOVERY_URL',
```
Comment (elias-summermatter) [inline config/create-gke-manifest.js:663]: Should we really disable oauth with google? im my opinion we should just delete OAUTH_WEPUBLISH... Variables

File: docker-compose.yml [hunk 2]
```diff
@@ -36,13 +36,6 @@ services:
       MEDIA_SERVER_INTERNAL_URL: http://media:4100
       MEDIA_SERVER_TOKEN: secret
       DATABASE_URL: postgresql://postgres@database:5432/wepublish?schema=public
-      OAUTH_GOOGLE_CLIENT_ID: 617896178757-i6ldn0nni9qtle8o6eu76lv93d78nvfi.apps.googleusercontent.com
```
Comment (elias-summermatter) [inline docker-compose.yml:39]: Same here just remove OAUTH_WEPUBLISH...

File: packages/api/src/runServer.ts [hunk 3]
```diff
@@ -119,22 +119,6 @@ export async function runServer() {
     })
   }
 
-  if (
-    process.env.OAUTH_WEPUBLISH_DISCOVERY_URL &&
-    process.env.OAUTH_WEPUBLISH_CLIENT_ID &&
-    process.env.OAUTH_WEPUBLISH_CLIENT_KEY &&
```
Comment (elias-summermatter) [inline packages/api/src/runServer.ts:125]: Here ists correct just OAUTH_WEPUBLISH removed

**Review by elias-summermatter — DISMISSED** [review]

File: config/create-gke-manifest.js (hunk 1 above)
Comment (tomaszdurka) [reply config/create-gke-manifest.js:663]: I left it removed as I will remove this deployment soon anyway. Noone uses this feature and for now local dev is enough.

File: docker-compose.yml (hunk 2 above)
Comment (tomaszdurka) [reply docker-compose.yml:39]: Readded `OAUTH_GOOGLE`.

**Review by Itrulia — CHANGES_REQUESTED** [review]
Just missing

```
  if (
    process.env.OAUTH_WEPUBLISH_DISCOVERY_URL &&
    process.env.OAUTH_WEPUBLISH_CLIENT_ID &&
    process.env.OAUTH_WEPUBLISH_CLIENT_KEY &&
    process.env.OAUTH_WEPUBLISH_REDIRECT_URL
  ) {
    oauth2Providers.push({
      name: 'wepublish',
      discoverUrl: process.env.OAUTH_WEPUBLISH_DISCOVERY_URL ?? '',
      clientId: process.env.OAUTH_WEPUBLISH_CLIENT_ID ?? '',
      clientKey: process.env.OAUTH_WEPUBLISH_CLIENT_KEY ?? '',
      redirectUri: [process.env.OAUTH_WEPUBLISH_REDIRECT_URL ?? ''],
      scopes: ['openid profile email']
    })
  }
  ```

and 

```
OAUTH_WEPUBLISH_DISCOVERY_URL=http://localhost:4200/.well-known/openid-configuration
OAUTH_WEPUBLISH_CLIENT_ID=wepublish
OAUTH_WEPUBLISH_CLIENT_KEY=secret
OAUTH_WEPUBLISH_REDIRECT_URL=http://localhost:4000/auth/wepublish
```

from example api

**Comment by tomaszdurka** [comment]
@Itrulia @antkiewiczk please review

**Review by Itrulia — APPROVED** [review]

**Pull request merged**